Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -927,13 +927,30 @@ class WebRTCClient: NSObject, @unchecked Sendable {
         var tracks = [Stream_Video_Sfu_Signal_TrackSubscriptionDetails]()
         let callParticipants = await state.callParticipants
 
+        /// Depending on the Device's thermal state we adapt the request participants resolution.
+        let scale: CGFloat = {
```

Comment:
Could we move this in an extension of the ThermalStateObserver? It's too much details for here.